src/main/java/Cow.java

```diff
@@ -0,0 +1,29 @@
+/* A sample class that implements the given interfaces.
+ */
+public class Cow implements Domesticatable, Tradable {
+    private String colour;
+
+    public Cow() {
+        this.colour = "brown";
+    }
+
+    public void Setcolour(String colour) {
```

Comment:
bad naming convention, should be "setColour"

Reply:
^